vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj

```diff
@@ -150,7 +150,7 @@
     <Import Project="$(SolutionDir)\packages\boost.1.68.0.0\build\boost.targets" Condition="Exists('$(SolutionDir)\packages\boost.1.68.0.0\build\boost.targets')" />
     <Import Project="$(SolutionDir)\packages\boost_date_time-vc141.1.68.0.0\build\boost_date_time-vc141.targets" Condition="Exists('$(SolutionDir)\packages\boost_date_time-vc141.1.68.0.0\build\boost_date_time-vc141.targets')" />
     <Import Project="$(SolutionDir)\packages\Microsoft.ChakraCore.vc140.1.11.9\build\native\Microsoft.ChakraCore.vc140.targets" Condition="Exists('$(SolutionDir)\packages\Microsoft.ChakraCore.vc140.1.11.9\build\native\Microsoft.ChakraCore.vc140.targets')" />
-    <Import Project="$(SolutionDir)\packages\ChakraCore.Debugger.0.0.0.34\build\native\Microsoft.ChakraCore.Debugger.targets" Condition="Exists('$(SolutionDir)\packages\ChakraCore.Debugger.0.0.0.34\build\native\Microsoft.ChakraCore.Debugger.targets')" />
+    <Import Project="$(SolutionDir)\packages\Microsoft.ChakraCore.Debugger.0.0.0.34\build\native\Microsoft.ChakraCore.Debugger.targets" Condition="Exists('$(SolutionDir)\packages\Microsoft.ChakraCore.Debugger.0.0.0.34\build\native\Microsoft.ChakraCore.Debugger.targets')" />
```

Comment:
OOI, it looks like we went through a cycle of dropping and re-adding the "Microsoft." prefix here. Do you know why?

Reply:
I only know there was reluctance because the `Microsoft` prefix is reserved.
@acoates-ms and @akrantz may have more insight.

Reply:
@acoates-ms Did you publish the ChakraCore-Debugger nuget manually, or was the ADO pipeline updated to use the public nuget feed instead of a private one?

Reply:
On second thought, looks like this line could be wrong. The `Microsoft`-prefixed package is not being used in master.

Reply:
@JunielKatarn That's what I was wondering :)